apps/meteor/app/api/server/v1/e2e.ts

```diff
@@ -230,3 +238,46 @@ API.v1.addRoute(
 		},
 	},
 );
+
+API.v1.addRoute(
+	'e2e.fetchUsersWaitingForGroupKey',
+	{
+		authRequired: true,
+		validateParams: isE2EFetchUsersWaitingForGroupKeyProps,
+	},
+	{
+		async get() {
+			if (!settings.get('E2E_Enable')) {
+				return API.v1.success({ usersWaitingForE2EKeys: {} });
```

Comment:
My idea was to prevent errors on the setInterval 🤔 but not sure if that's a valid concern as the e2e worker won't run unless e2e is enabled.

Other possibility is the interval running at the same time with the feature being disabled, which would make the interval trying to do a run and failing with an unhandled promise rejection. Do you think that's a valid concern?